src/parser/shadowlands/modules/items/dungeons/ShadowgraspTotem.tsx

```diff
@@ -14,7 +14,7 @@ import Statistic from 'parser/ui/Statistic';
 import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
 import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
 
-const ITEM: Item = ITEMS.SHADOWGRASP_TOTEM;
+const ITEM: Item = ITEMS.SHADOWGRASP_TOTEM as any;
```

Comment:
as any?

Reply:
Hmm, I initially meant it as a hack around inconsistencies of `Item` interfaces, but it might not even be needed seeing as this `ITEM` const is used only for its item ID. Will refactor